test/e2e/playwright/swap/support/signup-page.ts

```diff
@@ -0,0 +1,84 @@
+import { type Locator, type Page } from '@playwright/test';
```

Comment:
is this file duplicated? Leave just one and remove the pass and seed.
Actually folder pageObjects and support is are duplicated. I'd say remove support for coherence with mmi

Reply:
Yes you are right. Old code here. Forgot to delete